package.json

```diff
@@ -76,7 +77,7 @@
       }
     },
     "patchedDependencies": {
-      "@changesets/assemble-release-plan": "patches/@changesets__assemble-release-plan.patch"
+      "@changesets/assemble-release-plan@6.0.5": "patches/@changesets__assemble-release-plan.patch"
```

Comment:
What's the reason behind this change, is pkg-pr-new also using this (at a different version)

Reply:
Not sure, the patch broke on my local so I recreated the patch. If I exclude the version specifier it still doesn't work after the repatch.